azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -370,8 +368,7 @@ def remove(
         for name in certificate_names:
             if name not in [mapping["targetKey"] for mapping in secret_mapping]:
                 logger.warning(
-                    f"Certificate {name} not found in secretsync resource {secretsync_name}. "
-                    "Skipping removal..."
+                    f"Certificate {name} not found in secretsync resource {secretsync_name}. " "Skipping removal..."
```

Comment:
Same split string irregularity is present here

Reply:
😢I don't get why it reformat to this on save, will update

Reply:
even more strange, I do see this in 2 lines without modification😐
<img width="464" alt="image" src="https://github.com/user-attachments/assets/bc3d95b4-a18a-4f86-8fae-107144c9ae21" />

Reply:
Think it's because of string continuation on multiple lines, with formatter setting having a longer line length. And the formatter doesn't want to combine string content.